docs/books/admin_guide/09-backups.md

```diff
@@ -432,7 +432,7 @@ Verify 1/file2
 Verify 1/file3
 ```
 
-The verification with the `W` key cannot be done with a compressed archive. The key ++d++ must be used:
+The verification of compressed archives cannot be completed using the `W` key. The key `d` must be used:
 
```

Comment:
First, you are adding in passive voice phrasing here. Our goal is to get rid of passive voice as much as possible. Second, you are removing a markdown keyboard command, and replacing it with a single back-ticked letter (`d`). This should be ++"d"++ (if it is only recognized in lower-case) or as the author has (++d++) if upper-case is acceptable. In truth the `W` key of the author's original should be ++w++, which will render as a capital W.  An acceptable rewording of this pharse would be:
"You cannot verify the compressed archive with the ++w++ key. You must use the ++"d"++ key instead."

Reply:
If changed to "++++", it means that it is an interactive command typed by the user, but considering the context, the keyword used by the author is "key"

Reply:
I'll give you the `d`, but you *need* to change the passive voice phrasing to active voice. This was already done with the original document once, so I don't want to have to go through the document again and fix that. I've given you suggestions. You are free to use your own wording, provided it doesn't continue to introduce passive voice.